cfg/options.go

```diff
@@ -141,12 +141,14 @@ func WithFilesystem(fs string) Option {
 	}
 }
 
-// WithCandidateConfigFilename set the default candidate config filename
-// for your platform. Better to use only in the testing env where a
-// definite constant filename of config file on the device is helpful.
+// WithCandidateConfigFilename set the default candidate config filename for your platform. Better
+// to use only in the testing env where a definite constant filename of config file on the device
+// is helpful. If, however, you use this for "normal" operations, make sure you abort any loaded
+// configuration between if doing subsequent load operations as the candidate config will remain the
+// same when using this option!
 func WithCandidateConfigFilename(fn string) Option {
 	return func(p interface{}) error {
-		err := setPlatformAttr("CandidateConfigFilename", fn, p)
+		err := setPlatformAttr("candidateConfigFilename", fn, p)
```

Comment:
Can we set this struct attribute in this package? Last I checked it might crash at line num 109 as the attribute is not exported:
	fieldVal.Set(reflect.ValueOf(attrValue))

I might be completely wrong but this was the reason I had made this attribute uppercase as it was not accessible in this go package :). Please correct me if I am wrong.

Reply:
Ah good call. I think I let Goland auto refactor magic go a bit too crazy :P

Just pushed a fix I think/hope haha did it quick so let me know if I missed something!